### PR TITLE
Chore: Omnichannel endpoints e2e tests

### PR DESCRIPTION
**Pull request by KevLehman**

### CODE REVIEW

File: apps/meteor/server/models/raw/LivechatRooms.js [hunk 1]
```diff
@@ -996,6 +996,7 @@ export class LivechatRoomsRaw extends BaseRaw {
 			params.push({ $limit: options.limit });
 		}
 
+		console.log('params', JSON.stringify(params, null, 2));
```
Comment (murtaza98) [inline apps/meteor/server/models/raw/LivechatRooms.js:999]:
```suggestion
```
Oops

**Comment by murtaza98** [comment]
image
@KevLehman Is this file required?

**Comment by murtaza98** [comment]
@KevLehman FYI: left a few suggestions over here :wink: https://github.com/RocketChat/Rocket.Chat/pull/26412

**Comment by KevLehman** [comment]
Hey, yups. File is required to test : upload endpoint.

**Comment by murtaza98** [comment]
> Hey, yups. File is required to test : upload endpoint.

Alright, it looks good then :tada:

**Review by ggazzo — APPROVED** [review]

**Pull request merged**